processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -117,6 +164,32 @@ func (ctdp *cumulativeToDeltaProcessor) shouldConvertMetric(metricName string) b
 		(ctdp.excludeFS == nil || !ctdp.excludeFS.Matches(metricName))
 }
 
+func (ctdp *cumulativeToDeltaProcessor) convertFloatValue(id tracking.MetricIdentity, dp pmetric.HistogramDataPoint, value float64) (tracking.DeltaValue, bool) {
```

Comment:
Since this is only for HistogramDataPoints lets update to convey that.  Maybe `convertHistogramFloatValue`.  Same with `convertIntValue` below.